src/test/java/guitests/BoardAutoCreatorTest.java

```diff
@@ -65,6 +71,45 @@ public void boardAutoCreator_clickYesInSavePrompt_currentBoardSaved() {
         traverseMenu("Boards", "Open", "New Board");
         PlatformEx.waitOnFxThread();
         assertEquals(panelCount + 3, panelControl.getPanelCount());
+
+        traverseMenu("Boards", "Delete", "Milestones");
+        waitUntilNodeAppears("OK");
+        clickOn("OK");
+        traverseMenu("Boards", "Delete", "New Board");
+        waitUntilNodeAppears("OK");
+        clickOn("OK");
+    }
+
+    @Test
+    public void boardAutoCreator_clickNoInSavePrompt_currentBoardSaved() {
+        int panelCount = panelControl.getPanelCount();
+        assertEquals(0, panelControl.getNumberOfSavedBoards());
+
+        // create 3 new panels
+        pushKeys(KeyboardShortcuts.CREATE_RIGHT_PANEL);
+        pushKeys(KeyboardShortcuts.CREATE_RIGHT_PANEL);
+        pushKeys(KeyboardShortcuts.CREATE_RIGHT_PANEL);
+        PlatformEx.waitOnFxThread();
+        assertEquals(panelCount + 3, panelControl.getPanelCount());
+
+        // create milestones board
+        traverseMenu("Boards", "Auto-create", "Milestones");
+        PlatformEx.waitOnFxThread();
```

Comment:
From the docs in `PlatformEx`, the `waitOnFxThread` is used to block until the java fx events queue become empty. We should rename it so that the purpose is clearer?